### PR TITLE
Carry permissions over when excluding files 

### PR DESCRIPTION
**Pull request by arichiardi**
When excluding files from compiled zips in the plugin, permissions were lost. JSZip was not detecting or setting them from the old archive but this patch now correctly does it inside applyZipExclude.

### CODE REVIEW

File: serverless-cljs-plugin/serverless_lumo/build.cljs [hunk 1]
```diff
@@ -34,11 +37,12 @@
...
        (.pipe archiver output-stream)
        (doseq [d (:dirs zip-opts)]
-         (.directory archiver d))
+         (.directory archiver d #js {:mode 755}))
        (doseq [f (:files zip-opts)]
          (if (string? f)
-           (.file archiver f)
-           (.apply (.-file archiver) archiver (clj->js (seq f)))))
+           (.file archiver f #js {:mode 644})
+           (.apply (.-file archiver) archiver
```
Comment (moea) [inline serverless-cljs-plugin/serverless_lumo/build.cljs]: This line is quite hard to follow - if it's correct, can you bind the `update` expression in a `let` and use it by name inside .apply?

**Comment by arichiardi** [comment]
Hold on, this one apparently does not do what I want, not working recursively. I am [digging](https://github.com/archiverjs/node-archiver/issues/140#issuecomment-329333786).

File: serverless-cljs-plugin/serverless_lumo/build.cljs (hunk 1 above)
Comment (arichiardi) [reply serverless-cljs-plugin/serverless_lumo/build.cljs]: I sure can do that.

**Comment by arichiardi** [comment]
So this was the problem in #24

File: serverless-cljs-plugin/serverless_lumo/build.cljs [hunk 2]
```diff
@@ -22,7 +22,8 @@
     - https://archiverjs.com/docs/Archiver.html#file"
   [output-path zip-opts compiler-opts]
 
-  (let [archiver      (archiver "zip" #js {:zlib {:level 9}})
+  (let [archiver      (archiver "zip" #js {:zlib {:level 9}
+                                           :comment (str "Generated by serverless-cljs-plugin on" (.toISOString (js/Date.)))})
```
Comment (arichiardi) [inline serverless-cljs-plugin/serverless_lumo/build.cljs]: This is mirroring the one in `applyZipExclude` which is not always called, so here we take care of that.

Comment (moea) [reply serverless-cljs-plugin/serverless_lumo/build.cljs]: It's a very minor issue, but perhaps `const comment = ...` before the first use might be easier to follow than having very slightly different timestamps for each entry.

Comment (arichiardi) [reply serverless-cljs-plugin/serverless_lumo/build.cljs]: Ok will do

**Comment by arichiardi** [comment]
So this one should be good now, I addressed your valid point.
